django/utils/feedgenerator.py

```diff
@@ -223,7 +223,7 @@ def latest_post_date(self):
                     if latest_date is None or item_date > latest_date:
                         latest_date = item_date
 
-        return latest_date or datetime.datetime.now()
+        return latest_date or datetime.datetime.now(tz=timezone.utc)
```

Comment:
If I use `django.utils.timezone.now`, it will return a naive datetime object if it `USE_TZ = False`.

As per the discussion [at the ticket page](https://code.djangoproject.com/ticket/26653#comment:3), it was suggested we use `UTC`. 

You tell me, what do I do.